cpu/sam0_common/periph/dac.c

```diff
@@ -27,6 +27,10 @@
 
 #define DAC_VAL(in) (in >> (16 - DAC_RES_BITS))
 
+#ifndef CONFIG_SAM0_DAC_REFRESH
+#define CONFIG_SAM0_DAC_REFRESH 2
+#endif
+
```

Comment:
```suggestion
#ifndef CONFIG_SAM0_DAC_RUN_ON_STANDBY
#define CONFIG_SAM0_DAC_RUN_ON_STANDBY 0
#endif
```

otherwise this is impossible to find.